scripts/Utility.js

```diff
@@ -17,4 +17,24 @@ function formatHour(time) {
 		return time + 'a';
 	}
 	return (time - 12) + 'p';
-}
\ No newline at end of file
+}
+
+function getHolidays() {
```

Comment:
Kk has some holiday themed songs. I listed them as comments in utility.js because the song files still need to be added. Those are the only ones I could find, so I figured that's what the feature request was referring to. Is there special gameplay music on holidays?